CodeQL build failures

See, for example, https://github.com/DMTF/libspdm/actions/runs/19662654563/job/56312122900. This is supposedly with an MbedTLS build, so I am now sure why it is complaining about `cryptlib_openssl` not existing. https://github.com/DMTF/libspdm/pull/3367 probably triggered this issue.

```
-- Configuring done (3.5s)
CMake Error at os_stub/cryptlib_openssl/CMakeLists.txt:15 (add_dependencies):
  The dependency target "openssllib" of target "cryptlib_openssl" does not
  exist.


CMake Generate step failed.  Build files cannot be regenerated correctly.
-- Generating done (0.1s)
```

--- a/.github/workflows/codeql-analysis.yml
+++ b/.github/workflows/codeql-analysis.yml
@@ -36,7 +36,6 @@ jobs:
           - openssl
         arch:
           - x64
-          - ia32
         target:
           - Debug
           - Release

